src/python/espressomd/reaction_ensemble.pyx

```diff
@@ -400,12 +397,12 @@ cdef class WangLandauReactionEnsemble(ReactionAlgorithm):
 
     def add_collective_variable_degree_of_association(self, *args, **kwargs):
         """
-        Adds a reaction coordinate of the type degree of association.
+        Adds degree of association as the collective variable (reaction coordinate) for the Wang-Landau Reaction Ensemble.
```

Comment:
You can have multiple reaction coordinates of this type. Therefore it was previsously "adds a reaction coordinate of the type..." implying that you can have more than one reaction coordinates of this type.

Therefore I would explicitly state that you can have more than one reaction coordinate of this type.

Reply:
Agreed. I will add a note that several collective variables can be set simultaneously.